paig-server/pyproject.toml

```diff
@@ -51,7 +51,8 @@ dependencies = [
     "fasteners",
     "APScheduler",
     "pydantic[email]",
-    "posthog"
+    "posthog",
+    "paig-authorizer-core==0.0.1"
```

Comment:
I understand the suggestion to specify the dependency as paig-authorizer-core without a version number. However, I believe it's important to include the exact version (e.g., paig-authorizer-core==0.0.1) to ensure compatibility and avoid any potential issues that may arise from version changes in the future.

For example, if a user installs paig-server 0.0.2, and if we don't specify any specific version of paig-authorizer-core, they might inadvertently receive a newer version of the library that includes breaking changes or altered functionality and might not work with paig-server 0.0.2. This could lead to incompatibility issues where the behavior of my project does not work as intended with the latest version. By specifying the version, I can help ensure that users have the correct version that was tested and verified to work with my project, providing a smoother experience.